src/util/networkUtils.js

```diff
@@ -46,6 +46,10 @@ function writeNetworksJson (networkInfo, networkFilePath) {
 
 function toNetworkObject (networkInfo) {
   return networkInfo.reduce((acc, contract) => {
+    // Pruning unnecessary event data from network info
+    for (let networkId of Object.keys(contract.networks)) {
+      delete contract.networks[networkId]['events']
+    }
     acc[contract.name] = contract.networks
```

Comment:
mm, I think I made a mistake, cause with my tip, cause there's a nested loop there (reduce + for) so it would require a second reduce or create a new object. 

To keep things simple, I think it's fine to do it as your are doing.